middleware/integration_test/docker-compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3"
+services:
+  bitcoind:
+    image: nicolasdorier/docker-bitcoin:0.17.0
+    networks:
+      local:
+        ipv4_address: 10.10.0.10
```

Comment:
You actually don't need to define networks explicitly with docker compose version > 3 anymore
All you need is `depends_on:` and the docker compose creates networks between those containers for you
All you then need to do to connect to that service over the internal docker network is provide the service name instead of IP, e.g bitcoind:18443

Reply:
Could not get electrs running without defining the network explicitly. Since it is running as is, lets keep this on the back of our mind, once we add more functionality.